README.rst

```diff
@@ -15,7 +15,7 @@ Parsl can be used to implement various parallel computing paradigms:
 * Dynamic workflows in which the workflow is determined during execution
 * Interactive parallel programming through notebooks or another interactive mechanism
 
-The latest Parsl version available on PyPi is v1.1.0.
+The latest Parsl version available on PyPi is v1.3.0.
```

Comment:
not any more! there's an automatic release process and nothing to update any version numbers outside of the release. see eg https://github.com/Parsl/parsl/issues/2463